package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "retry-my-fetch",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
Since we add new functionality it would be better if we bump it up to a minor version, something like `1.4.0`